server/routes/github.js

```diff
@@ -0,0 +1,87 @@
+const express = require("express");
```

Comment:
going off of the server.js code, maybe it would be better if we enforce semicolons? 😄